### DISCUSSION TITLE
Still not get the benefits after half a month

### DISCUSSION THREAD

**Discussion by tlc314960** [community/community#146349] [GitHub Education]
### Hi everyone,

Hi there, I have got approval since 22 Nov, but I still haven't got the benefits email. It's almost half a month, anything wrong with my application?

**Comment by tlc314960** [comment]
<img width="309" alt="Snipaste_2024-12-05_22-58-30" src="https://github.com/user-attachments/assets/867e40bc-26f8-464e-bab5-89507dd2ce52">

**Comment by davevad93** [comment]
Hi @tlc314960 ,

You should have access to your benefits. Check if you can redeem your offers by going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and any offer, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.
